pyanaconda/core/configuration/system.py

```diff
@@ -187,3 +187,8 @@ def can_use_driver_disks(self):
     def supports_web_ui(self):
         """Can we run Web UI on this system?"""
         return self._is_boot_iso or self._is_live_os
+
+    @property
+    def supports_compositor_keyboard_layout_shortcut(self):
+        """Does the compositor supports keyboard layout options correctly?"""
```

Comment:
supports -> support

Reply:
I'll change the comment but I'm leaving it in the method name as it is used in the method above the same way.